android/src/main/java/nl/fhict/happynews/android/activity/MainActivity.java

```diff
@@ -88,6 +100,35 @@ public void onScrolled(RecyclerView recyclerView, int dx, int dy) {
         });
     }
 
+
+    /**
+     * Shows an error message when the app is not connected to the internet.
+     */
+    public void notConnectedError() {
+        ConnectivityManager connectivityManager = (ConnectivityManager) getSystemService(Context.CONNECTIVITY_SERVICE);
+        NetworkInfo activeNetwork = connectivityManager.getActiveNetworkInfo();
+
+        boolean isConnected = activeNetwork != null && activeNetwork.isConnected();
```

Comment:
Is er een reden om dit niet als conditions van de volgende if statement neer te zetten?